packages/headless/src/controllers/folded-result-list/headless-folded-result-list.ts

```diff
@@ -172,6 +185,13 @@ export function buildFoldedResultList(
           ] as string
         )
       );
+      dispatch(logShowMoreFoldedResults(collection.result));
+    },
+    logShowMoreFoldedResults: (result) => {
```

Comment:
Related to my above comment, if the event is now dispatched when loading the collection, this function now seems superfluous and could be removed.